environment

```diff
@@ -39,6 +39,7 @@ druid_metadata_storage_connector_password=FoolishPassword
 
 druid_coordinator_balancer_strategy=cachingCost
 
+druid_worker_capacity=4
```

Comment:
Trying to keep us from "for this notebook, change the startup parameters X, Y, and Z to "a","b", and "c".... 
I reduced the memory used by each Peon in a prior PR as well so that we could do more tasks. It doesn't hurt anything to have more until you use the Peons and they get instantiated. 
I'm erring on the side of "no friction" which means no parameters on startup... we may need to change this, but for now I think it is a good approach to get to "Druid is easy" :-D